src/jdk.javadoc/share/classes/jdk/javadoc/doclet/Reporter.java

```diff
@@ -94,6 +94,12 @@
      * The positions are all 0-based character offsets from the beginning of content of the file.
      * The positions should satisfy the relation {@code start <= pos <= end}.
      *
+     * @implNote
+     * This implementation throws {@code UnsupportedOperationException}.
```

Comment:
I'd suggest rephrasing this as follows: "This implementation always throws..." both here and in the respective CSR.